CHANGELOG.md

```diff
@@ -2,6 +2,15 @@
 
 ## Unreleased
 
+- Allow non-`'static` event sources and callbacks, so they can hold references
+  to other values. **Breaking Changes**:
+  - `LoopHandle::insert_source` now returns the `Token` which can be used to
+    `enable`, `disable`, `update`, ... the source just like before.
+  - `LoopHandle::with_source` was removed (see below).
+  - `LoopHandle::remove` doesn't return the event source any more (see below).
+  - `LoopHandle::register_dispatcher` can be used in place of
+    `LoopHandle::insert_source` when the source needs to be accessed after
```

Comment:
Since API is changing to use a dispatcher, instead of `with_source`, I'd encourage to write more verbose changelog on that and not `(see below)`, since it's a bit unclear about it here.

Reply:
Added some details. Hope this better now.